Makefile

```
@@ -489,10 +494,31 @@ verify-links:
 	fi;
 .PHONY: verify-links
 
-verify: verify-gofmt verify-codegen verify-crds verify-helm-schemas verify-helm-charts verify-deploy verify-examples verify-govet verify-helm-lint verify-links
+# $1 - source file
+# $2 - client directory
+# $3 - client name
+define generate-swagger-client
+	GOPATH=$(GOPATH) $(GO) run "$(SWAGGER)" generate client --name "$(3)" --spec "$(1)" --target "$(2)"
+
+endef
+
+update-manager-client:
+	$(call generate-swagger-client,${MANAGER_CLIENT_SPEC},${MANAGER_CLIENT_GEN_DIR},'manager')
+.PHONY: update-manager-client
+
+verify-manager-client: tmp_dir := $(shell mktemp -d)
+verify-manager-client: MANAGER_CLIENT_TMP_DIR := "$(tmp_dir)/${MANAGER_CLIENT_GEN_DIR}"
+verify-manager-client:
+	cd $(tmp_dir) && $(GO) mod init $(GO_MODULE) && mkdir -p "${MANAGER_CLIENT_GEN_DIR}"
```

```suggestion
mkdir -p "$(tmp_dir)/$(MANAGER_CLIENT_GEN_DIR)"
cd $(tmp_dir) && $(GO) mod init $(GO_MODULE)
```
`-p` is error prone, this way it's at least more safely bounded to the tmp_dir if a substitution was to fail

---

also `"${MANAGER_CLIENT_GEN_DIR}"` should be substituted in make, not bash (bracket types)

(applies globally)